docs/appendix/contributing.md

```diff
@@ -311,6 +311,8 @@ For regular videos that should use the video shortcode as follows:
 
 {{<video webm_src="/heart.webm" mp4_src="/heart.mp4" alt="A robot drawing a heart" poster="/general/heart.jpg">}}
 
+Make sure your `webm` and `mp4` files are less than 1MB.
```

Comment:
sorry no, it doesn't. I thought you were adding it as a gif, that is my bad. Sorry!

Reply:
Will close